apps/wallet/src/background/Transactions.ts

```diff
@@ -65,8 +65,8 @@ class Transactions {
             throw new Error('Transaction signature is empty');
         }
         if (tx) {
-            if (!txResult || !('transaction' in txResult)) {
-                throw new Error(`Transaction result is empty`);
+            if (!txResult || !('digest' in txResult)) {
```

Comment:
after this change, `transaction` field is by default empty unless `options.showInput` is true